### ISSUE TITLE
Bug: Z459 should accept Ellipsis 

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#586]
# Bug report

## What's wrong

False positive for `./returns/maybe.pyi:79:5: Z459 Found complex default value`
It should accept `ast.Ellipsis` as a default value, since it is a standard for `.pyi` files.

## How is that should be

```python
@final
class _Nothing(Maybe[Any]):
    _inner_value: None

    def __init__(self, inner_value: None = ...) -> None:
        ...
```

**Comment by phoolish-philomath** [comment]
I can take care of this (:

**Comment by sobolevn** [comment]
@phoolish-philomath awesome!
